source/_posts/azure-purview.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Azure Purview
+date: 2021-03-22 19:38:02
+tags:
+---
+
+If you don't know what kind of data you have and where it resides, it's difficult to use it. Azure Purview can help with that! 
```

Comment:
it's difficult to use it >> it is difficult to use your data effectively
Azure Purview can help with that! >> Azure Purview is here to solve that problem!